### ISSUE TITLE
Feature request: open right links/text in sequence

### ISSUE THREAD

**Issue by sleek22** [harytfw/GlitterDrag#16]
Right now, when choosing "Right" to open links or text, the order looks like this:

A 3 2 1 B C

So if a person wanted to search text in say, Google Search, then Google Images, and then Wikipedia, the child tabs line up in reverse as Wikipedia, then Google Images, then Google Search.

A more natural sequence would be in the order the search was done, as follows:

A 1 2 3 B C

This follows the same custom that tab browsing follows, i.e., child tabs open to the right adjacent to their parent tabs, and in the order in which they are opened.

**Comment by harytfw** [comment]
I agree with you idea.



What do you think if we close child tabs  1 2 3 , the order becomes
A B C
should we change current tab to `A` or don't handle it( current tab becomes `B` , follow browser's setting)

**Comment by sleek22** [comment]
I think it should go back to A.

Say we are reading an article and we encounter a word or phrase we don't understand. We stop to search the meaning of it, then we want to continue reading the original article.

In the rare case where we decide we don't need to come back to the original article, we open the word or phrase in the background and when done reading the original article, close that tab. In most browser setups, that moves us to the next tab to the right, which in this case is the child tab we opened previously.

In my particular case, I use Tab Utilities Fixed which always goes back to the parent Tab. Tab Mix Plus which is used by most users only does this for one child tab. If more than one child tab is closed, it would go to Tab B. (Which is the main reason I use Tab Utilities Fixed to always go back to the parent tab).

So in my case, default behavior is to always go back to A, except when that tab has been closed. Since most users use Tab Mix Plus, it gives them more options as they can always override by closing the parent tab, which means the browser then defaults to the B tab.
